pkgs/by-name/et/ethercat/kernel.nix

```diff
@@ -0,0 +1,199 @@
+{
+  autoreconfHook,
+  pkg-config,
+  stdenv,
+  pkgs,
+  lib,
+  kernel,
+}:
+let
+  deviceKernelCompatabilities = {
```

Comment:
We must comment the source of this

Reply:
Resolved with

https://github.com/NixOS/nixpkgs/blob/14a1b28727fac1532a2b21269c0f5f0d317e4405/pkgs/by-name/et/ethercat/kernel.nix#L13-L14